src/audio/copier/copier.c

```diff
@@ -899,8 +899,18 @@ static int copier_prepare(struct comp_dev *dev)
 			return ret;
 	}
 
+	if (dev->ipc_config.type == SOF_COMP_DAI) {
```

Comment:
`switch()`

Reply:
switch will cover 3 cases, host/dai/gtw, anything not covered?